apps/web/src/server/lib/nunjucks-filters/key-dates-property.js

```diff
@@ -0,0 +1,65 @@
+/** @type {Record<string, string>} */
+const propertyToName = {
```

Comment:
I hope the mapping is correct. 

Thanks to @chrisdalykinandcarta  the names were straightforwarding, but I could have done some mistake anyway.